framework/src/modules/chain/block_processor_v2.js

```diff
@@ -184,6 +184,16 @@ class BlockProcessorV2 extends BaseBlockProcessor {
 
 		this.init.pipe([() => this.bftModule.init()]);
 
+		this.deserialize.pipe([
+			({ block }) => this.blocksModule.deserialize(block),
+			(_, updatedBlock) => this.bftModule.deserialize(updatedBlock),
```

Comment:
Looks clean with an underscore but might be confusing with _ lodash we often input as underscore.
Possible nasty bug? 🗡

Reply:
I think _ is common approach and we generally don't use `_` as lodash anymore as we agreed on importing specific lodash functions instead of the whole module.